src/components/Search/CompareOverTime.tsx

```diff
@@ -1,25 +1,106 @@
+import { useState } from 'react';
+
+import { Grid } from '@mui/material';
+import Divider from '@mui/material/Divider';
+import { useLocation } from 'react-router-dom';
+import { Form } from 'react-router-dom';
+import { style } from 'typestyle';
+
 import { useAppSelector } from '../../hooks/app';
 import { Strings } from '../../resources/Strings';
-import { CompareCardsStyles } from '../../styles';
+import { CompareCardsStyles, SearchStyles } from '../../styles';
+import { RevisionsList, Repository } from '../../types/state';
+import CompareButton from './CompareButton';
+import FrameworkDropdown from './FrameworkDropdown';
+import SearchOverTime from './SearchOverTime';
 
 const strings = Strings.components.searchDefault;
+const stringsNew =
+  Strings.components.searchDefault.overTime.collapsed.revisions;
+
+interface CompareWithTimeProps {
+  isEditable: boolean;
+  baseRevs: RevisionsList[];
+  newRevs: RevisionsList[];
+  baseRepos: Repository['name'][];
+  newRepos: Repository['name'][];
+}
+
+function CompareOverTime({ isEditable }: CompareWithTimeProps) {
+  const [expanded, setExpanded] = useState(false);
 
-function CompareOverTime() {
   const mode = useAppSelector((state) => state.theme.mode);
   const styles = CompareCardsStyles(mode);
+  const dropDownStyles = SearchStyles(mode);
+  const search = useAppSelector((state) => state.search);
+  const searchResults = search.new.searchResults;
+  //temporary hash to hide the component until functionality is complete
+  const location = useLocation();
+  const hash = location.hash;
+
+  //temporary styles to hide the component based on hash
+  const containerStyles = {
+    container: style({
+      display: hash === '#comparetime' ? 'block' : 'none',
+    }),
+  };
+
+  const bottomStyles = {
+    container: style({
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'flex-end',
+    }),
+  };
+
+  const toggleIsExpanded = () => {
+    setExpanded(!expanded);
+  };
+
   return (
-    <div
-      className={`compare-card-container compare-card-container--time ${styles.container}`}
+    <Grid
+      className={`wrapper--overtime ${styles.wrapper} ${containerStyles.container}`}
     >
-      <div className={`compare-card-text ${styles.cardText}`}>
-        <div className='compare-card-title'>{strings.overTime.title}</div>
-        <div className='compare-card-tagline'>{strings.overTime.tagline}</div>
+      <div
+        className={`compare-card-container compare-card-container--${
+          expanded ? 'expanded' : 'hidden'
+        } ${styles.container}`}
+        onClick={toggleIsExpanded}
+        data-testid='time-state'
+      >
+        <div className={`compare-card-text ${styles.cardText}`}>
+          <div className='compare-card-title'>{strings.overTime.title}</div>
```

Comment:
Random thought for accessibility: this should be a header
=> https://mozilla-hub.atlassian.net/browse/PCF-392

(but I'm fine if we change both of them later)

Reply:
I can do this now. No prob.

Reply:
I've change them now since it makes the tests easier.